apps/code-examples/src/app/code-examples/indicators/icon/basic/icon-demo.component.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyIconHarness } from '@skyux/indicators/testing';
+
+import { IconDemoComponent } from './icon-demo.component';
+import { IconDemoModule } from './icon-demo.module';
+
+describe('Basic icon', async () => {
+  async function setupTest(): Promise<SkyIconHarness> {
+    const fixture = TestBed.createComponent(IconDemoComponent);
+    const loader = TestbedHarnessEnvironment.loader(fixture);
+    const iconHarness = await loader.getHarness(
+      SkyIconHarness.with({
+        dataSkyId: 'icon-demo',
+      })
+    );
+    fixture.detectChanges();
```

Comment:
Detecting changes should be governed by each `it` block (sometimes it's needed to run `expect` before the first change detection cycle). I would remove this line, and return the `fixture` from the `setupTest` function.
```
return { fixture, iconHarness };

// ...

it('should ...', async () => {
  const { fixture, iconHarness } = await setupTest();

  fixture.detectChanges();
});
```